OOBE/5.7.0/Groups/Profiles/groups.dcl

```diff
@@ -339,243 +339,5 @@ data member_group super_administrator {
 patch member_group super_administrator_patch {
```

Comment:
idk if somethings is depending on this or if i can just remove it?

Reply:
It is still changing the name, but you should probably remove the `roles` property entirely, otherwise it will just remove all the roles.